### PR TITLE
docs: update README and add aka refered doc

### PR DESCRIPTION
**Pull request by spouliot**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

* README.md was out-of-date.
* Exception could show `https://aka.platform.uno/skia-macos` but the doc is not available

## What is the new behavior?

* Updated READMD.md
* Doc to match `https://aka.platform.uno/skia-macos`

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by spouliot** [comment]
@jeromelaban / @agneszitte once merged, could you link https://aka.platform.uno/skia-macos with the new document ? thanks!

File: doc/articles/features/using-skia-macos.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+uid: Uno.Skia.macOS
+---
+
+# Using the Desktop (Skia) on macOS
+
+## Metal (Hardware Accelerated) Rendering
+
+All recent Mac computers support Metal. It has been required to run macOS 10.14 (Mojave). As such macOS hardware-accelerated [Metal](https://developer.apple.com/metal/) rendering is enabled by default.
+
+However it's possible that under certain circumstances, like virtualization (including [GitHub CI](https://github.com/actions/runner-images/issues/1779)), Metal might not be available to applications. In this case the host will **automatically** fall back to a software-based rendering.
```
Comment (agneszitte) [inline doc/articles/features/using-skia-macos.md:11]:
```suggestion
However, it's possible that under certain circumstances, like virtualization (including [GitHub CI](https://github.com/actions/runner-images/issues/1779)), Metal might not be available to applications. In this case, the host will **automatically** fall back to a software-based rendering.
```

File: doc/articles/features/using-skia-macos.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+> [!NOTE]
+> Some Mac models released before 2012 did not support Metal. See Apple's support document [Support for Metal on Mac, iPad, and iPhone](https://support.apple.com/en-us/102894) for the list of devices that supports Metal.
```
Comment (agneszitte) [inline doc/articles/features/using-skia-macos.md:14]:
```suggestion
> Some Mac models released before 2012 did not support Metal. See Apple's support document [Support for Metal on Mac, iPad, and iPhone](https://support.apple.com/en-us/102894) for the list of devices that support Metal.
```

File: src/Uno.UI.Runtime.Skia.MacOS/README.md [hunk 3]
```diff
@@ -1,23 +1,17 @@
...
 
 ## Requirements
 
-* Minimum: macOS 10.14
+* Minimum OS version: same as the [dotnet version used](https://learn.microsoft.com/en-us/dotnet/core/install/macos)
 
 ## Pros
 
-* Less dependencies
+* Faster startup: Less dependencies
```
Comment (agneszitte) [inline src/Uno.UI.Runtime.Skia.MacOS/README.md:9]:
```suggestion
* Faster startup: Fewer dependencies
```

**Review by Youssef1313 — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**